Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -76,6 +76,23 @@ configuration:
   - Incoming And Outgoing
   required: false
   type: 15
+- additionalinfo: Select this checkbox to retrieve comments and work notes without accessing the `sys_field_journal` table.
+  display: Use Display Value
+  name: use_display_value
+  required: false
+  type: 8
+- additionalinfo: 'Select the date format of your ServiceNow instance. Mandatory when using the `Use Display Value` option. To find the date format of your instance see https://docs.servicenow.com/bundle/tokyo-platform-administration/page/administer/time/task/t_PersonalizeTheSystemDateFormat.html'
```

Comment:
please move the explanation of where to get the formats to the readme, and leave a here a direction to the place in the readme. this is because additional info is a hover popup. so it is not clickable and you cannot copy from it.